gensrc/thrift/PlanNodes.thrift

```diff
@@ -757,6 +759,15 @@ struct THdfsScanNode {
 
     // hive colunm names in ordinal order.
     7: optional list<string> hive_column_names;
+
+    // hive table name it scans
+    8: optional string hive_table_name;
```

Comment:
is it better named as table_name? since we may have iceberg and hive table for hdfs_scan_node?

Reply:
Good idea.